bg3-modders-multitool/bg3-modders-multitool/Services/PathHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using bg3_modders_multitool.Properties;
```

Comment:
Place using statements within the class and add a summary (check other services)